cmd/minikube/cmd/mount.go

```diff
@@ -46,6 +50,23 @@ var mountCmd = &cobra.Command{
 			fmt.Fprintln(os.Stderr, errText)
 			os.Exit(1)
 		}
+		if args[0] == "kill" {
```

Comment:
I am not sure what you are referring to.  This is so that a user who wants to stop the mount process that is spawned from `minikube start` has a way to do so without having to restart minikube.
```
minikube start
#don't want the mount
minikube mount kill
```